instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/grpcext/_interceptor.py

```diff
@@ -41,41 +39,67 @@ class _StreamClientInfo(
 ):
     pass
 
+def _get_metadata_timeout(**kwargs):
+    metadata = kwargs.get("metadata")
+    timeout = kwargs.get("timeout")
+    return metadata, timeout
+
 
 class _InterceptorUnaryUnaryMultiCallable(grpc.UnaryUnaryMultiCallable):
     def __init__(self, method, base_callable, interceptor):
         self._method = method
         self._base_callable = base_callable
         self._interceptor = interceptor
 
-    def __call__(self, request, timeout=None, metadata=None, credentials=None):
+    def __call__(
+        self,
+        request,
+        **kwargs,
```

Comment:
What do you think of doing something like this instead of `_get_metadata_timeout()` throughout?

```py
def __call__(
    self,
    request,
    *,
    metadata=None,
    timeout=None,
    **kwargs
):
    ...
```

Reply:
I don't see any advantage on using `kwargs` here if every call will necessarily require `timeout` and `metadata`.